### PR TITLE
fix: css-loader 1.0 removed alias

### PR DESCRIPTION
**Pull request by clarkdo**
Fixes issues with @<organization>/<package> for plugins and more.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Related to https://github.com/nuxt/nuxt.js/issues/3723
Because `css-loader` has removed `options.alias` and is using `loader-utils` for `url()` now.
So this is a breaking change for `<url>`, users need to prefix `~` when they want to use `alias`:

> Now: please use `url(~assets)`(**recommend**), `url(~@/assets)` or `url(~~/assets)`
> Do not use: `url(~/assets)`

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: lib/builder/webpack/base.js [hunk 1]
```diff
@@ -101,19 +101,15 @@ export default class WebpackBaseConfig {
...
-      ),
-      [this.options.dir.static]: path.join(
-        this.options.srcDir,
-        this.options.dir.static
-      )
+      '~': path.join(srcDir),
+      '~~': path.join(rootDir),
+      '@': path.join(srcDir),
+      '@@': path.join(rootDir),
+      [assetsDir]: path.join(srcDir, assetsDir),
```
Comment (pi0) [inline lib/builder/webpack/base.js]: I prefer the previous way of not de-structuring. It is somehow more clear to read.

**Review by pi0 — APPROVED** [review]

**Comment by husayt** [comment]
This breaks vuetify in a la-carte-mode.
https://vuetifyjs.com/en/guides/a-la-carte

it is complaining at this line in latest 1.2.4 version of 
```
require('../../../src/stylus/components/_app.styl');
```
in this file
...\node_modules\vuetify\es5\components\VApp\VApp.js

**Comment by clarkdo** [comment]
Could you provide a repo ? @husayt

**Comment by husayt** [comment]
Hi @clarkdo 
here is repo with simple demo.
https://github.com/husayt/nuxt2-vuetify

latest commit works with     "nuxt-edge": "2.0.0-25607354.1e02195",
the commit before then with "nuxt": "2.0.0" is broken.

Thanks

**Comment by husayt** [comment]
@clarkdo did the example I provide make sense, or do you need more info?

**Comment by clarkdo** [comment]
@husayt Sorry about that I have some personal stuff recently, I’ll look into it immediately when available.

**Comment by aldarund** [comment]
@husayt @clarkdo  use transpile instead of config.externals & nodeExternals and it will work. I have same vuetify a la carte setup and it work fine.
PS in vuetify 1.3 it wont be needed due to auto a la carte

**Comment by husayt** [comment]
@aldarund thanks for a tip.

Here is a working config from me:

```
 build: {
    transpile: [/^vuetify/],
    babel: {
      plugins: [
        [
          "transform-imports",
          {
            vuetify: {
              transform: "vuetify/es5/components/${member}",
              preventFullImport: true
            }
          }
        ]
      ]
    },
```

**Pull request merged**